tools/vscode-azurewebpubsub/src/workflows/service/openLiveTraceTool/OpenLiveTraceToolStep.ts

```diff
@@ -0,0 +1,52 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardExecuteStep } from "@microsoft/vscode-azext-utils";
+import { type IPickServiceContext } from "src/workflows/common/contexts";
+import * as vscode from 'vscode';
+import { createAzureApiClient, createEndpointFromHostName, localize } from '../../../utils';
+import * as jwt from "jsonwebtoken";
+
+export class OpenLiveTraceToolStep extends AzureWizardExecuteStep<IPickServiceContext> {
+    public priority: number = 110;
+
+    public async execute(context: IPickServiceContext, progress: vscode.Progress<{ message?: string | undefined; increment?: number | undefined }>): Promise<void> {
+        if (!context.serviceName || !context.resourceGroupName || !context.subscription) {
+            throw new Error(localize('invalidIPickServiceContext', `Invalid IPickService Context, subscription {0}, service name {1}, resource group {2}`, context.subscription?.subscriptionId, context.serviceName, context.resourceGroupName));
+        }
+
+        const client = await createAzureApiClient([context, context.subscription]);
+        progress.report({ message: localize('openingLiveTraceTool', 'Opening LiveTrace Tool, please wait...') });
+
+        const resource = (await (client.webPubSub.get(context.resourceGroupName, context.serviceName)));
+        const keys = await (client.webPubSub.listKeys(context.resourceGroupName, context.serviceName));
+        if (!resource.hostName) {
+            throw new Error(localize('invalidResource', `Invalid resource, hostName {0}`, resource.hostName));
+        }
+        const endpoint = createEndpointFromHostName(resource.hostName);
+        /* eslint-disable */
+        const accessToken = jwt.sign({}, keys.primaryKey, {
```

Comment:
shall we by default support VSCodeCredential connect?

Reply:
It seems other extensions don't use `VisualStudioCodeCredential`. Let's follow them?
Ref:
https://github.com/microsoft/vscode-azurecontainerapps
https://github.com/Microsoft/vscode-azurefunctions
https://github.com/microsoft/vscode-azurespringcloud
https://github.com/microsoft/vscode-azureresourcegroups